locales/ko.yaml

```diff
@@ -6,111 +6,111 @@ page-title:
   home: 홈
   song: 곡 정보
   songs: 곡 목록
-  gallery: 단위인정
-  about: 사이트 정보
-  bug-report: 문의
+  gallery: 트랙 목록
+  about: 정보
+  bug-report: 문제 보고
   source-code: 소스 코드
 
 # General UIs
 ui:
-  ok: OK
+  ok: 확인
   cancel: 취소
   close: 닫기
   search: 검색
-  goBack: 뒤로
-  all: 전체
+  goBack: 뒤로 가기
+  all: 모두
   default: 기본값
   sort-asc: 오름차순
   sort-desc: 내림차순
 
 # Terminologies
 term:
-  category: 카테고리
+  category: 범주
   title: 제목
-  artist: 아티스트명
+  artist: 아티스트
   version: 버전
   releaseDate: 수록일
   bpm: BPM
   minBPM: 최소 BPM
   maxBPM: 최대 BPM
-  sheets: 보면
-  type: 타입
+  sheets: 악보
+  type: 유형
   difficulty: 난이도
-  level: 레벨
-  minLevel: 최소 레벨
-  maxLevel: 최대 레벨
-  internalLevel: 보면상수
-  noteDesigner: 보면 제작자
+  level: 수준
+  minLevel: 최소 수준
+  maxLevel: 최대 수준
+  internalLevel: 내부 수준
+  noteDesigner: 노트 제작자
   totalNotes: 전체 노트 수
-  region: 국가
+  region: 지역
 
 # Descriptions & Tooltips
 description:
-  pending: 처리 중
-  loading: 데이터 로딩 중…
+  pending: 보류 중
+  loading: 데이터 불러오기 중…
   newSong: 신곡!
-  unlockNeeded: 해금 필요
-  unavailableInRegion: '{region} 미수록'
-  copied: 클립보드에 복사되었습니다!
-  invalidNoteCounts: 노트 수가 유효하지 않습니다
-  filterResultEmpty: 필터 값을 만족하는 보면을 찾을 수 없습니다.
-  myListEmpty: 보면을 우클릭하거나 길게 눌러 마이리스트에 추가하여 모아볼 수 있습니다.
-  drawPoolEmpty: 리스트가 비어있습니다.
+  unlockNeeded: 해금이 필요한 곡
+  unavailableInRegion: '현재 {region}에서 사용할 수 없음'
+  copied: 클립보드에 복사했습니다!
+  invalidNoteCounts: 잘못된 노트 수
+  filterResultEmpty: 필터를 만족하는 악보를 찾을 수 없습니다.
+  myListEmpty: 악보를 마우스 오른쪽 버튼으로 클릭하거나 길게 눌러 내 목록에 추가하면 여기에서 선택한 악보를 볼 수 있습니다..
+  drawPoolEmpty: 현재 악보 풀이 비어 있습니다.
 
 # Pages
 page:
   index:
-    description: 좋아하는 리듬게임의 노래정보를 한 곳에서 모아보세요!
+    description: 좋아하는 음악 게임의 곡을 이제 모두 하나로 검색하세요!
```

Comment:
This sentence doesn't make sense.. I think my version is better. I might be biased though